Gemfile

```diff
@@ -4,6 +4,7 @@ ruby '2.2.3' # this should match `.ruby-version` and docs/setup.md
 gem 'active_model_serializers'
 gem 'activeadmin', github: 'activeadmin'
 gem 'acts_as_list'
+gem 'acts-as-taggable-on', '~> 3.4'
```

Comment:
that's a cargo-cult line from the acts-as-taggable README. Since it was important to the author, I assumed it should be important to us?